### ISSUE TITLE
Trying to build for Linux, getting error on Win32

### ISSUE THREAD

**Issue by cheapy** [adamyg/grief#1]
I used to have Brief, and wrote lots of macros for it and loved it...

Now I'm trying to build Grief for Linux.  Maybe I'm just missing something simple...

Here is what I did:
./configure_new
no errors

make
loading:  ./makelib.in
make -C libmisc
make[1]: Entering directory '/home/bobc/source/grief/libmisc'
gcc -g -O2  --param max-inline-insns-single=1200 -pthread -g -Og  -I/home/bobc/source/grief/include  -DHAVE_CONFIG_H  -o /home/bobc/source/grief/objects.gcc/debug/libmisc/edthreads_win32.o -c edthreads_win32.c
In file included from edthreads_win32.c:42:
/home/bobc/source/grief/include/edthreads.h:126:15: error: expected ';' before 'extern'
 __CBEGIN_DECLS
               ^
               ;
/home/bobc/source/grief/include/edthreads.h:128:1:
 extern struct timespec      timespec_diff(const struct timespec end, const struct timespec start);
 ~~~~~~         
/home/bobc/source/grief/include/edthreads.h:131:13: error: expected ';' before 'extern'
 __CEND_DECLS
             ^
             ;
make[1]: *** [Makefile:216: /home/bobc/source/grief/objects.gcc/debug/libmisc/edthreads_win32.o] Error 1
make[1]: Leaving directory '/home/bobc/source/grief/libmisc'
make: *** [Makefile:375: /home/bobc/source/grief/lib.gcc/debug/libmisc] Error 2

**Comment by ircole** [comment]
I'm having the same problem.  Is there a resolution?

**Comment by cheapy** [comment]
It looks like tickets for this should be on Sourceforge, not Github

this might be the same issue

https://sourceforge.net/p/grief/tickets/13/

**Comment by adamyg** [comment]
shall review, thx

**Comment by adamyg** [comment]
https://ci.appveyor.com/project/adamyg/grief-ubuntu,  appveyor clean build, same for Centos (local primary host); assume toolchain issue and/or glibc version issue. Can you fwd results of your configure exec, report result of 'ldd --version' and value HAVE_THREADS_H within include/config.h.

**Comment by cheapy** [comment]
I put them all in a text file for you and added a couple other files that are likely relevant.

Thanks for helping :)
[requested-info.txt.zip](https://github.com/adamyg/grief/files/6215815/requested-info.txt.zip)

**Comment by adamyg** [comment]
corrected, header include order; GLIBC 2.28-10 related

**Comment by ircole** [comment]
Thanks for the change.  It fixed the problem on Linux Mint 20.1 that I 
was having.

**Comment by cheapy** [comment]
I was able to compile successfully now as well.  Thanks Much for your help!
